### ISSUE TITLE
Copy/paste does not work (again?)

### ISSUE THREAD

**Issue by ygoe** [xps/VSTextMacros#37]
I've seen several issues about this but they're all old and closed. Maybe this is a new cause?

Recording a sequence with Ctrl+C (or Ctrl+Ins) and Ctrl+V (or Shift+Ins) doesn't do the copy and/or paste on playback. This defeats a bit the use of macros that need to reuse other text in a line and apply the same operation on many lines.

Reverting to Notepad++ again.

**Comment by xps** [comment]
@ygoe Do you have the Productivity Power Tools extension installed?

**Comment by ygoe** [comment]
No. That's the all-in-one monster, right? I didn't like that. Rather pick the stuff I need.

**Comment by ygoe** [comment]
Oh, I forgot all the data. This is Visual Studio 2019 v16.7.5, extension version 1.15.

**Comment by xps** [comment]
@ygoe Nothing that would be used to copy formatted code from VS ? (Like "Copy to HTML")
This has been none to break the recording of copy/paste operations.

In any case, copy/paste records and replays fine for me with version 1.15 on VS 2019 16.7.2.

If you want to troubleshoot, I would start by disabling all other extensions and see if that fixes it.

**Comment by ygoe** [comment]
No, I can't find any extensions that deal with the copy or paste function in the editor. Great, so I have to go the long route again. Well, let's see when I find the time for that.

**Comment by PaltryProgrammer** [comment]
greetings kind regards
attached please find GIF demonstrate use of clipboard failure . please note PPT "Copy as HTML" is installed but not enabled .
thank you kindly
![Clipboard Failure](https://github.com/xps/VSTextMacros/assets/48535920/0df3df44-8c37-4dab-b0a9-fcca231d280f)

**Comment by xps** [comment]
Hi, and thank you for reporting this.

Unfortunately, I'm not sure why it's not working on your setup.
It's definitely working on my Visual Studio 2022 with a few extensions installed.
I can only guess that one of your extensions are interfering.

One thing you could do, to troubleshoot, is to disable all the extensions and test it again.
If it works, then you can reenable half of the extensions and that way find by dichotomy which one is the problematic one.

I am sorry that I cannot help further.
